code/game/objects/items/storage/book.dm

```diff
@@ -50,6 +50,8 @@ GLOBAL_LIST_INIT(bibleitemstates, list("bible", "koran", "scrapbook", "bible", "
 /obj/item/storage/book/bible/attack_self(mob/living/carbon/human/H)
 	if(!istype(H))
 		return
+	if(!H.can_read(src))
```

Comment:
I feel like this actually shouldn't have the check, since setting the icon of the bible isn't exactly *reading* it, and is kind of a meta-action.